cpp/tests/utilities/column_utilities.cu

```diff
@@ -371,5 +372,28 @@ void print(cudf::column_view const& col, std::ostream &os, std::string const& de
   os << to_string(col, delimiter);
 }
 
+bool validate_host_masks(std::vector<bitmask_type> const& expected_mask,
+                         std::vector<bitmask_type> const& got_mask,
+                         size_type number_of_elements) {
+
+    std::vector<cudf::size_type> mask_index(expected_mask.size());
+    std::iota(mask_index.begin(), mask_index.end(), 0);
+    size_type bitmask_size = sizeof(bitmask_type) * 8;
```

Comment:
Never compute this manually. https://github.com/rapidsai/cudf/blob/branch-0.13/cpp/include/cudf/utilities/bit.hpp#L32